src/script/script.h

```diff
@@ -412,6 +413,32 @@ bool GetScriptOp(CScriptBase::const_iterator& pc, CScriptBase::const_iterator en
 /** Serialized script, used inside transaction inputs and outputs */
 class CScript : public CScriptBase
 {
+private:
+    inline void AppendDataSize(const size_t size)
+    {
+        if (size < OP_PUSHDATA1) {
+            insert(this->end(), static_cast<value_type>(size));
+        } else if (size <= 0xff) {
+            insert(this->end(), OP_PUSHDATA1);
+            insert(this->end(), static_cast<value_type>(size));
+        } else if (size <= 0xffff) {
+            insert(this->end(), OP_PUSHDATA2);
+            value_type _data[2];
+            WriteLE16(_data, size);
+            insert(this->end(), std::cbegin(_data), std::cend(_data));
```

Comment:
nit: Don't like the `_`-prefix of `_data`, but it's from the original. `_`-prefixes otherwise signify something internal or reserved.
Since you are touching all lines, maybe switch it to `data`/`data2`/ `data16` + `data`/`data4`/`data32`?

Reply:
or just `data` since they don't overlap.